app/src/main/java/com/example/myapplication/TaskActivity.kt

```diff
@@ -5,35 +5,31 @@ import android.os.Bundle
 import com.google.android.material.floatingactionbutton.FloatingActionButton
 import com.google.android.material.snackbar.Snackbar
 
-import androidx.appcompat.app.AppCompatActivity
-import androidx.appcompat.widget.Toolbar
-
-import android.view.View
+import android.widget.Toolbar
 import android.view.Menu
 import android.view.MenuItem
-import androidx.recyclerview.widget.LinearLayoutManager
-import androidx.recyclerview.widget.RecyclerView
-
-class TaskActivity : AppCompatActivity() {
+import androidx.fragment.app.FragmentActivity
 
-    private lateinit var recyclerView: RecyclerView
+class TaskActivity : FragmentActivity() {
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
         val toolbar = findViewById<Toolbar>(R.id.toolbar)
-        setSupportActionBar(toolbar)
+        setActionBar(toolbar)
 
         val fab = findViewById<FloatingActionButton>(R.id.fab)
         fab.setOnClickListener { view ->
             Snackbar.make(view, "Replace with your own action", Snackbar.LENGTH_LONG)
                     .setAction("Action", null).show()
         }
 
-        val data = listOf("dishes", "groceries", "trash", "clean room")
-        recyclerView = findViewById(R.id.task_recycler_view)
-        recyclerView.layoutManager = LinearLayoutManager(this)
-        recyclerView.adapter = TaskAdapter(this, data)
+        //initialize fragment WHY IS THIS NOT WORKING
+        val fragmentManager = supportFragmentManager
```

Comment:
If I comment out these fragment lines the rest of the activity works.  With them I get a whole white screen.